ogr/services/gitlab.py

```diff
@@ -216,7 +216,12 @@ def add_issue_labels(self, issue_id, labels) -> None:
         raise NotImplementedError()
 
     def get_pr_list(self, status: PRStatus = PRStatus.open) -> List["PullRequest"]:
-        mrs = self.gitlab_repo.mergerequests.list(order_by="updated_at", sort="desc")
+        # Gitlab API has status 'opened', not 'open'
```

Comment:
Hmm, it looks like inconsistency on our site -- we have `closed`, but `open`. (And it also looks like GitHub does not have `open` either.)

I would go for changing the enum value, but we need to check the usage in other projects:

- [ ] packit
- [ ] packit-service
- [ ] release-bot

Reply:
OK, it looks like we can break release-bot and packit with the change => Let's stay with your fix. (And probably use the same fix in the GitHub as well.)

Reply:
I've created a new issue for that:
https://github.com/packit-service/ogr/issues/193